.github/DISCUSSION_TEMPLATE/q-a.yml

```diff
@@ -0,0 +1,41 @@
+title: "<Your question in one sentence (e.g., How do I preview documents with Quarto?)>"
+body:
+  - type: markdown
+    attributes: 
+      value: |
+        Make sure to check [Quarto Issues](https://github.com/quarto-dev/quarto-cli/issues) and [Q&A GitHub Discussions](https://github.com/quarto-dev/quarto-cli/discussions/categories/q-a) first to see if your question/report has already been discussed.
+        If it has, you can add a comment to the existing discussion instead of creating a new one.
+
+        After checking, if you are not sure that your question/report has already been discussed, you can still create a new discussion. 😉
```

Comment:
Fixed this and added:

> Finally, try to describe the best you can what you want to achieve or what is your issue, especially by providing a complete self-contained reproducible example (as demonstrated in the below form).
> This will help us to understand your question and answer it.

See https://github.com/quarto-dev/quarto-cli/pull/5531/commits/18a3f8ff707d7a4ec4a0164c3e87bf2e36295175